vdb-bench-assembly/plugins/vdb-bench-dataservice/dsTestController.js

```diff
@@ -13,8 +13,8 @@
     function DSTestController($scope, DSSelectionService) {
         var vm = this;
 
-        vm.dsDeployInProgress = false;
-
+        vm.dsDeployInProgress = true;
```

Comment:
Why is this set to true to being with? Seems it only needs to be true when a deployment is in progress.